app/journey/JourneyController.scala

```diff
@@ -69,7 +69,7 @@ class JourneyController @Inject() (actions:        Actions,
 
   def updateWithPcipalData(): Action[ChargeRefNotificationPcipalRequest] = Action.async(parse.json[ChargeRefNotificationPcipalRequest]) { implicit request =>
     val notification: ChargeRefNotificationPcipalRequest = request.body
-    logger.info(s"Updating journey with Pcipal data (upon ChargeRefNotification) [paymentItemId:${notification.paymentItemId.value}] [PCIPalSessionId:${notification.PCIPalSessionId.value}] [HoD:${notification.HoD.toString}]")
+    logger.info(s"Updating journey with Pcipal data [paymentStatus: ${notification.Status}] [paymentItemId:${notification.paymentItemId.value}] [PCIPalSessionId:${notification.PCIPalSessionId.value}] [HoD:${notification.HoD.toString}]")
```

Comment:
Could you add extra logs few lines below when we actually find the journey and update it. Please log also the journeyId, so it will be easier to find that journey has been updated when querying only by journeyId.

Reply:
Updated and added a few more logs in, which I'll use in the kibana graphs. There were a couple of places I couldn't access journeyId, but I've instead (tried) to use a combination of journeyId/pcipalSessionId/paymentItemId, so we should be able to piece journeys together a bit nicer

Reply:
It might be worth raising a small ticket to implement a journey logger object like in other repos, to have this stuff done automagically by the methods